index.html

```diff
@@ -789,13 +789,11 @@ <h3>Presentations</h3>
         </p>
 
         <p>
-A <a>verifiable presentation</a> expresses data from one or more
-<a>verifiable credentials</a>, and is packaged in such a way that the
-authorship of the data is <a>verifiable</a>. If <a>verifiable credentials</a>
-are presented directly, they become <a>verifiable presentations</a>. Data
-formats derived from <a>verifiable credentials</a> that are cryptographically
-<a>verifiable</a>, but do not of themselves contain
-<a>verifiable credentials</a>, might also be <a>verifiable presentations</a>.
+A <a>verifiable presentation</a> can express data from multiple
+<a>verifiable credentials</a> and contain arbitrary additional data encoded as
+JSON-LD. They are used by a <a>holder</a> to present <a>claims</a> to a
+<a>verifier</a>. It is also possible to present <a>verifiable credentials</a>
+directly.
```

Comment:
What does presenting VCs "directly" mean?

Reply:
If "directly" means that a VC is a VP, then we need an example.

If we just say that one does not need to use a VP as an envelope for a VC (which I think is what the sentence means), then we have to define under which circumstances VPs are useful at all. IMO, presenting VCs should be the default case, and only if x, y and z matter to the implementer, then you can use a VP to express x, y, z. Does this make sense?

Reply:
In the present VC directly case, we should also define what the holder is. The holder might be the credential subject, or just unknown, or inferred by the proof and respective securing mechanism.

Reply:
I don't disagree with much of what you say, I'm wondering if this needs to happen in this PR.

Reply:
n.b., the section in which this language is found is a non-normative use cases summary that I believe should simply be removed #1169

Reply:
Got it, that makes sense.